webpack/webpack.config.babel.js

```diff
@@ -0,0 +1,64 @@
+import autoprefixer from 'autoprefixer';
+import colorLighten from 'less-color-lighten';
+import path from 'path';
+
+function absPath() {
+  let args = [].slice.apply(arguments);
+  args.unshift(__dirname, '..');
+
+  return path.resolve.apply(path.resolve, args);
+}
```

Comment:
Could you use this instead?
https://nodejs.org/api/path.html#path_path_normalize_path